splinepy/io/gismo.py

```diff
@@ -0,0 +1,345 @@
+        # offset of 1)
+        interface_array = np.hstack(
+                (
+                        con_spline_id_start[start_order] + index_offset,
+                        con_face_id_start[start_order] + 1,
+                        con_spline_id_end[end_order] + index_offset,
+                        con_face_id_end[end_order] +1 ,
+                        # This is the orientation:
+                        np.repeat(
+                                np.arange(multipatch.para_dim,
+                                          dtype=np.int32).reshape(1, -1),
+                                start_order.size,
+                                axis=0
+                        ),
+                        np.ones(
+                                (start_order.size, multipatch.para_dim),
+                                dtype=np.int32
+                        )
+                )
+        )
+        interface_data.text = '\n'.join(
+                [' '.join([str(xx) for xx in x]) for x in interface_array]
+        )
+
+    # Start extracting all boundaries
+    # Boundaries are stored as patch-id (global) face-id (local)
+    boundary_spline, boundary_face = np.where(multipatch.interfaces < 0)
+    if boundary_spline.size > 0:
+        boundary_data = ET.SubElement(multipatch_element, 'boundary')
+        boundary_data.text = '\n'.join(
+                [
+                        str(sid + index_offset) + ' ' + str(bid+1)
+                        for (sid, bid) in zip(boundary_spline, boundary_face)
+                ]
+        )
+    ###
+    # Boundary conditions
+    ###
+    # If there are multiple boundary IDs write them into boundary conditions
+    # and fill with dummy values
+    # Note here, that the boundary conditions are always stored in the local
+    # enumeration system of them multipatch
+    boundary_condition_list = multipatch.boundaries
+    if len(boundary_condition_list) > 1:
+        bcs_data = ET.SubElement(
+                xml_data, 'boundaryConditions', multipatch=str(0), id=str(1)
+        )
+        bcs_data.insert(
+                0, ET.Comment(text="Please fill boundary conditions here")
+        )
+        for bc_data_i in boundary_condition_list:
+            bc = ET.SubElement(bcs_data, 'bc', type="Dirichlet", unknown="0")
+            bc.text = '\n'.join(
+                    [
+                            str(sid) + ' ' + str(bid+1)
+                            for (sid, bid) in zip(bc_data_i[0], bc_data_i[1])
+                    ]
+            )
+
+    ###
+    # Individual spline data
+    ###
+    for id, spline in enumerate(multipatch.splines):
+        if not issubclass(type(spline), Spline):
+            raise ValueError(
+                    "One of the splines handed to export is not a valid spline"
+                    " representation"
+            )
+
+        # Transform bezier types, as they are not supported in gismo
+        if spline.whatami.startswith("Bezier"):
```

Comment:
```suggestion
        if spline.name.startswith("Bezier"):
```
Just a hint, not critical at all